src/styles/components/menu-tabbed/styles.less

```diff
@@ -11,11 +11,13 @@
   }
 
   .menu-tabbed-container-fixed {
+    // force the menu to it's own composite layer to fix a bug in
+    // Firefox 63 and below on Windows and Linux
+    transform: translateZ(0);
 
     .menu-tabbed-vertical {
       height: 100%;
       position: fixed;
-      top: 135px;
```

Comment:
What is this change helping with?

Reply:
This number wasn't accurate, so the tab border was positioned incorrectly. It's much worse in those older FF versions.

![Screen Shot 2019-06-04 at 2 49 09 PM](https://user-images.githubusercontent.com/2313998/58905545-036c5500-86d8-11e9-9b44-743f4dca4cc1.png)